doc/user/content/stylesheet.md

```diff
@@ -68,11 +68,17 @@ This is a note.
 This is a warning.
 {{</ warning >}}
 
-### `beta` shortcode
+### `private-preview` shortcode
 
-{{< beta >}}
-This is a beta notice.
-{{</ beta >}}
+{{< private-preview >}}
+This is a private preview notice.
+{{</ private-preview >}}
+
+### `public-preview` shortcode
+
+{{< public-preview >}}
+This is a public preview notice.
```

Comment:
nit: Based on the current rendering logic, the text inside should just name the feature (and hence should not name with a dot). The same applies for the `public-preview` shortcode.